lamindb/models/feature.py

```diff
@@ -460,35 +531,35 @@ class Feature(SQLRecord, CanCurate, TracksRun, TracksUpdates):
         :class:`~lamindb.Schema`
             Feature sets.
 
-    Example:
+    Example::
 
         A simple `"str"` feature.
```

Comment:
If you remove the `>>>`, you need to add a double colon after a "A simple `"str"` feature.